### DISCUSSION TITLE
Feature request: Private outgoing IP for dedicated servers on Umbraco Cloud

### DISCUSSION THREAD

**Discussion by EPP-KMD** [umbraco/Umbraco.Cloud.Issues#836] [Ideas]
# Issue description
## What
Umbraco cloud projects with a dedicated server should have a way to get a static outgoing IP that is not shared with projects from other organizations.

## Why
My organization is paying for a dedicated Umbraco Cloud server hosting the corporate website. However, Umbraco Cloud says the server's IP may change at any moment, and when using the static outgoing IP the IP range is shared for everyone's cloud project in the region. This means authenticating by IP is a massive safety risk. As a client of Umbraco Cloud with a dedicated server, it would bring me a lot of value to have a way to authenticate requests from that server by IP without allowing every other Umbraco Cloud project in the region the same access, and not have to worry about it randomly stopping working without warning because my outgoing IP has been changed.

**Comment by meyntony** [comment]
Static IP only means that the IPs assigned will not change, but other customer's Cloud projects will still have the same static IP range if they have the feature enabled. https://docs.umbraco.com/umbraco-cloud/set-up/project-settings/external-services#enabling-static-outbound-ip-addresses 

We do not currently support dedicated/unique static IP's for a given project, which is what this feature request is about, correct?

**Reply by EPP-KMD** [reply]
Yes, that is correct
